examples/lines_bars_and_markers/fill_demo.py

```diff
@@ -16,3 +16,22 @@
 ax.fill(x, y, zorder=10)
 ax.grid(True, zorder=5)
 plt.show()
+
+"""
+========================
```

Comment:
You need to merge these two docstrings into something coherent at the beginning of the file. Just smooshing the separate files together is no good.